package.json

```diff
@@ -159,7 +159,7 @@
     "client:e2e": "node test/client/e2e/runner.js",
     "client:test": "npm run client:unit && npm run client:e2e",
     "start": "gulp run:dev",
-    "postinstall": "bower --config.interactive=false install -f; gulp build; npm run client:build"
+    "postinstall": "bower --config.interactive=false install -f && gulp build && npm run client:build"
```

Comment:
Forgot to leave it out of the commit. ';' doesn't work on windows so I changed it to '&&'